test/testfilter.jl

```diff
@@ -0,0 +1,35 @@
+include("../src/miscsetup.jl")
+include("../src/filter.jl")
+include("../src/track.jl")
+include("../src/model.jl")
+
+using Base.Test
+
+sz = (1024, 1024)
+nframe = 25
+@time imgs0 = zeros(sz...,  nframe)
+@time for k in 1:nframe
+    imgs0[:,:,k] = convert(Matrix{Float64}, Images.load(joinpath(Pkg.dir("TrackRoots"), "test", "testimage", "$k.TIF")))
+end
+imgs = normalize!(imgs0[1:768, :, :])
```

Comment:
To normalise or not to normalise here only affects the error estimate in `track` which we do not use yet.